sc-models/knowledge-base/metasystem_decomposition.scs

```diff
@@ -11,39 +11,38 @@ metasystem_decomposition
 	[OSTIS Metasystem decomposition]
 	(* <- lang_en;; *);;
 
-@arc1 = (metasystem_decomposition -> rrel_1: knowledge_base_Metasystem);;
+@arc1 = (metasystem_decomposition -> rrel_1: metasystem_knowledge_base);;
 
-@arc2 = (metasystem_decomposition -> problem_solver_Metasystem);;
+@arc2 = (metasystem_decomposition -> metasystem_problem_solver);;
 @arc1 => nrel_basic_sequence: @arc2;;
 
-@arc3 = (metasystem_decomposition -> user_interface_Metasystem);;
+@arc3 = (metasystem_decomposition -> metasystem_user_interface);;
 @arc2 => nrel_basic_sequence: @arc3;;
 
 @arc4 = (metasystem_decomposition -> subsystem_Metasystem);;
 @arc3 => nrel_basic_sequence: @arc4;;
 
 rrel_last ~> @arc4;;
 
-user_interface_Metasystem
+metasystem_user_interface
 => nrel_main_idtf:
 	[Пользовательский интерфейс Метасистемы OSTIS]
 	(* <- lang_ru;; *);
 	[OSTIS Metasystem user interface]
 	(* <- lang_en;; *);;
 
-
-user_interface_Metasystem
+metasystem_user_interface
 <- sc_model_of_interface;
 => nrel_decomposition: user_interface_Metasystem_decomposition;;
 
-problem_solver_Metasystem
+metasystem_problem_solver
 <- sc_model_of_problem_solver;
 => nrel_decomposition: problem_solver_Metasystem_decomposition;;
 
-knowledge_base_Metasystem
+metasystem_knowledge_base
 <- sc_model_of_knowledge_base;
 => nrel_decomposition: knowledge_base_Metasystem_decomposition;;
 
-concept_subsystems_set
-<- concept_subsystems_set;
+metasystem_subsystems
+<- concept_reusable_embedded_ostis_system;
```

Comment:
It is not reusable_embedded_ostis_system, it is a set of it. `concept_subsystems_set` was ok